src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
+                    // generate fasta files for mismatching sequences
+                    GATKPath ref1Path = refPair.getRef1();
+                    GATKPath ref2Path = refPair.getRef2();
+
+                    String sequenceInRef1Name = ref1Path.toPath().getFileName() + "." + sequenceName;
+                    String sequenceInRef2Name = ref2Path.toPath().getFileName() + "." + sequenceName;
+
+                    File ref1TempFastaOutput = IOUtils.createTempFile(sequenceInRef1Name, ".fasta");
+                    File ref2TempFastaOutput = IOUtils.createTempFile(sequenceInRef2Name, ".fasta");
+
+                    GATKPath ref1Fasta = generateFastaForSequence(ReferenceDataSource.of(ref1Path.toPath(), true), sequenceName, new GATKPath(ref1TempFastaOutput.toString()));
+                    GATKPath ref2Fasta = generateFastaForSequence(ReferenceDataSource.of(ref2Path.toPath(), true), sequenceName, new GATKPath(ref2TempFastaOutput.toString()));
+
+                    // pass fastas into mummer, get back a vcf and add to list
+                    MummerExecutor executor = new MummerExecutor();
+                    logger.info("Running mummer alignment on sequence " + sequenceName);
+                    File tempSnpsDirectory = IOUtils.createTempDir("tempsnps");
+                    File mummerOutput = executor.executeMummer(ref1Fasta.toPath().toFile(), ref2Fasta.toPath().toFile(), tempSnpsDirectory);
+                    logger.info("Finished running mummer alignment on sequence " + sequenceName);
+                    snpsFiles.add(mummerOutput);
+                }
+            }
+            // merge individual snps files
+            File snps = IOUtils.createTempFile(String.format("%s_%s", refPair.getRef1AsString(), refPair.getRef2AsString()), ".snps");
+            /*new File(baseComparisonOutputDirectory.toPath().toString(), String.format("%s_%s.snps", refPair.getRef1AsString(), refPair.getRef2AsString()));*/
+            try (PrintWriter writer = new PrintWriter(snps)) {
+                for (File file : snpsFiles) {
+                    try (XReadLines reader = new XReadLines(file)) {
+                            for (String line : reader) {
+                                writer.write(line + "\n");
+                            }
+                    } catch (IOException e) {
+                        throw new UserException("Error merging show-snps outputs.", e);
+                    }
+                }
+            }
+            catch(IOException e){
+                throw new UserException("Error writing show-snps output file.");
+            }
+            File vcf = new File(baseComparisonOutputDirectory.toPath().toString(), String.format("%s_%s.vcf", refPair.getRef1AsString(), refPair.getRef2AsString()));
+            try(XReadLines reader = new XReadLines(snps);
+                VariantContextWriter writer = createVCFWriter(vcf);
+                ReferenceDataSource source = ReferenceDataSource.of(refPair.getRef1().toPath())){
```

Comment:
Add a blank line after the `{` for readability here, since this try-with-resources is a bit crowded